index.html

```diff
@@ -0,0 +1,162 @@
+                    <button class="task-list-item__button task-list-item__button--delete">&#10060</button>
+                  </li> 
+                  <li class="task-list__item">
+                    <button class="task-list-item__button task-list-item__button--is-done">&check;</button> 
+                    
+                  <label class="task-list-item__content">
+                      Meet with friends
+                  </label>
+                  <button class="task-list-item__button task-list-item__button--delete">&#10060</button>
+                </li> 
+                <li class="task-list__item">
+                  <button class="task-list-item__button task-list-item__button--is-done">&check;</button> 
+                  
+                <label class="task-list-item__content">
+                    Meet with friends
+                </label>
+                <button class="task-list-item__button task-list-item__button--delete">&#10060</button>
+              </li> 
+              <li class="task-list__item">
+                <button class="task-list-item__button task-list-item__button--is-done">&check;</button> 
+                
+              <label class="task-list-item__content">
+                  Meet with friends
+              </label>
+              <button class="task-list-item__button task-list-item__button--delete">&#10060</button>
+            </li>  
+            <li class="task-list__item">
+              <button class="task-list-item__button task-list-item__button--is-done">&check;</button> 
+              
+            <label class="task-list-item__content">
+                Meet with friends
+            </label>
+            <button class="task-list-item__button task-list-item__button--delete">&#10060</button>
+          </li> 
+          <li class="task-list__item">
+            <button class="task-list-item__button task-list-item__button--is-done">&check;</button> 
+            
+          <label class="task-list-item__content">
+              Meet with friends
+          </label>
+          <button class="task-list-item__button task-list-item__button--delete">&#10060</button>
+        </li> 
+        <li class="task-list__item">
+          <button class="task-list-item__button task-list-item__button--is-done">&check;</button> 
+          
+        <label class="task-list-item__content">
+            Meet with friends
+        </label>
+        <button class="task-list-item__button task-list-item__button--delete">&#10060</button>
+      </li> 
+      <li class="task-list__item">
+        <button class="task-list-item__button task-list-item__button--is-done">&check;</button> 
+        
+      <label class="task-list-item__content">
+          Meet with friends
+      </label>
+      <button class="task-list-item__button task-list-item__button--delete">&#10060</button>
+    </li> 
+    
+        </ul>
+      </form>
+
+      <div class="task-form__wrapper">
+      <form class="task-form task-form__done">
```

Comment:
Technically, I dont see a reason to use `form` element in here - you will eventually display a **list** of files, so `form` => `ul` :P